tests/src/Interop/StructMarshalling/PInvoke/MarshalStructAsParamDLL.h

```diff
@@ -63,8 +63,6 @@ struct INNER2 // size = 12 bytes
     LPCSTR f3;
 };
 
```

Comment:
If the static assert doesn't hold, then the comment about 'size = 12 bytes' should definitely be removed since it is wrong. If seems like you want the condition to be 12 or 16 no?

Reply:
Yeah. I thought I removed that comment. My mistake.